docker/.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
This is great you added a `.dockerignore` as none of our ualbertalib images currently has this. So great job :+1: .

However we should add a few more things to this:
```
log/*
coverage
.travis.yml
pkg
.bundle
*.md
docs
```

Maybe bin folder too?